src/waveform/renderers/waveformwidgetrenderer.h

```diff
@@ -39,6 +40,8 @@ class WaveformWidgetRenderer {
 
     inline const char* getGroup() const { return m_group;}
     const TrackPointer getTrackInfo() const { return m_pTrack;}
+    // Get cue at a point on the waveform widget.
```

Comment:
```suggestion
    /// Get cue at a point on the waveform widget.
```
We're using `///` for documentation so this is picked up by IDE tooltips and Doxygen